README.md

```diff
@@ -7,64 +7,111 @@
 An [Apache 2.0](https://github.com/allenai/allennlp/blob/master/LICENSE) NLP research library, built on PyTorch,
 for developing state-of-the-art deep learning models on a wide variety of linguistic tasks.
 
-## Running AllenNLP
+## Installation
 
-The preferred way to install AllenNLP is via `pip`.
+The preferred way to install AllenNLP is via `pip`.  Just run `pip install allennlp` in your Python 3.6 environment and you're good to go!
 
-tldr; Just run `pip install allennlp` in your python 3.6 environment and you're good to go!
-If you need pointers on setting up a python 3.6 environment, see below.
+If you need pointers on setting up a Python 3.6 environment or would like to install AllenNLP using a different method, see below.
 
-_Just want AllenNLP models running as a service via Docker? `docker run --rm  -p 8000:8000 allennlp/allennlp:v0.4.2 python -m allennlp.run serve`_
+### Installing via pip
 
-### Setting up a virtual environment
+#### Setting up a virtual environment
 
-[Conda](https://conda.io/) can be used set up a virtual environment
-with the version of Python required for AllenNLP and in which you can
-sandbox its dependencies. If you already have a python 3.6 environment
-you want to use, you can skip to the 'installing via pip' section.
+[Conda](https://conda.io/) can be used set up a virtual environment with the
+version of Python required for AllenNLP.  If you already have a Python 3.6
+environment you want to use, you can skip to the 'installing via pip' section.
 
 1.  [Download and install Conda](https://conda.io/docs/download.html).
 
 2.  Create a Conda environment with Python 3.6
 
-    ```
+    ```bash
     conda create -n allennlp python=3.6
     ```
 
 3.  Activate the Conda environment. You will need to activate the Conda environment in each terminal in which you want to use AllenNLP.
 
-    ```
+    ```bash
     source activate allennlp
     ```
 
-### Installing via pip
+#### Installing the library and dependencies
 
-1.  Install AllenNLP.
+Installing the library and dependencies is simple using `pip`.
 
-    ```
-    pip install allennlp
-    ```
+```bash
+pip install allennlp
```

Comment:
do we want to specify the current version here just to be explicit

or is it too hard to keep that up to date?

Reply:
I don't know.  While I like being explicit, I expect our users to just run `pip install allennlp`.  It's not hard to keep up to date, but I worry it may fall behind.

What do you think?

Reply:
I'm going to merge and build on top of this PR.  I think it's best to leave the version out, but feel free to re-open if you start having a strong perspective on this--I don't have one.